pennylane/devices/tests/test_measurements.py

```diff
@@ -17,6 +17,7 @@
 import numpy as np
 import pytest
 from flaky import flaky
+from scipy.sparse import coo_matrix
```

Comment:
Is there any reason why we opt for the COO format, rather than CSR here? Shouldn't matter much for tests, but I wonder if this is because of uses elsewhere.

Reply:
Good question, here I'm just following the convention :thinking: Maybe @soranjh has some thoughts?

Reply:
@mlxd @trbromley, A quick benchmark showed that the COO format was a bit more advantageous over other formats available in scipy.sparse for performing efficient qchem simulations using sparse Hamiltonians. That is why we chose it.